src/briefcase/exceptions.py

```diff
@@ -241,7 +241,9 @@ def __init__(self, install_hint=""):
             f"""\
 Unable to install requirements. This may be because one of your
 requirements is invalid, or because pip was unable to connect
-to the PyPI server.{install_hint}
+to the PyPI server.
+
+{install_hint}
```

Comment:
It's wild to me that this part was pre-existing - we thought about the need for an install hint... and then never used it 🤣 

However, the original form is preferable here; if there's no install hint, we don't want extra blank lines after the exception message. If the install hint needs some extra lines (and I agree it does), those should be factored into the hint itself.
```suggestion
to the PyPI server.{install_hint}
```